infra/Pos/NtpCheck.hs

```diff
@@ -63,3 +69,20 @@ timeDifferenceWarnInterval = fromIntegral (Infra.ccTimeDifferenceWarnInterval in
 
 timeDifferenceWarnThreshold :: HasInfraConfiguration => Microsecond
 timeDifferenceWarnThreshold = fromIntegral (Infra.ccTimeDifferenceWarnThreshold infraConfiguration)
+
+type NtpDiffVar = TVar Microsecond
```

Comment:
I would expect this variable to contain `NtpStatus`.

Reply:
@int-index Why? `NtpDiffVar` does contain the time difference in Microsecond only. But it says nothing about the status. See mkNtpDiffVar and endpoint handler `localTimeDifference`.

Reply:
@int-index Ok, I see. Check last commit, pls.